src/agent/inspectordebugapi.ts

```diff
@@ -394,11 +460,13 @@ export class InspectorDebugApi implements debugapi.DebugApi {
       } else {
         const frame = callFrames[0];
         const evaluatedExpressions = breakpoint.expressions.map(function(exp) {
-          const result = state.evaluate(exp, frame, that.v8Inspector);
+          const result = state.evaluate(exp, frame, that.v8Inspector, true);
```

Comment:
Can you add a comment as to why we are passing in `true` here?

Reply:
Done.